hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/recon/TestReconTasks.java

```diff
@@ -166,5 +179,64 @@ public void testMissingContainerDownNode() throws Exception {
                   0, 1000);
       return (allMissingContainers.isEmpty());
     });
+    IOUtils.closeQuietly(client);
+  }
+
+  @Test
+  public void testEmptyMissingContainerDownNode() throws Exception {
+    ReconStorageContainerManagerFacade reconScm =
+        (ReconStorageContainerManagerFacade)
+            cluster.getReconServer().getReconStorageContainerManager();
+
+    StorageContainerManager scm = cluster.getStorageContainerManager();
+    PipelineManager reconPipelineManager = reconScm.getPipelineManager();
+    PipelineManager scmPipelineManager = scm.getPipelineManager();
+
+    // Make sure Recon's pipeline state is initialized.
+    LambdaTestUtils.await(60000, 5000,
```

Comment:
Can we reduce the interval to 2s or even 1s?

```suggestion
    LambdaTestUtils.await(60000, 2000,
```